### DISCUSSION TITLE
Vulnerability database update interval

### DISCUSSION THREAD

**Discussion by laurentiusoica** [aquasecurity/trivy-operator#1243] [Help & Support] [answered]
When Trivy is executed by the trivy operator, is the vulnerability database ever refreshed? If it is, is the refresh interval configurable?

**Comment by chen-keinan** [comment, marked as answer]
@laurentiusoica trivy db update occur every 6h , currently not configurable 

in addition there is a [TTL](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L59) (24h) for each report when it exceeded the resource get re-scan again
